component/src/test/java/io.siddhi.extension.store.cosmosdb/DeleteFromCosmosTableTest.java

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package io.siddhi.extension.store.cosmosdb;
+
+import io.siddhi.core.SiddhiAppRuntime;
+import io.siddhi.core.SiddhiManager;
+import io.siddhi.core.exception.SiddhiAppCreationException;
+import io.siddhi.core.stream.input.InputHandler;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+
+public class DeleteFromCosmosTableTest {
+
+    private static final Log log = LogFactory.getLog(DeleteFromCosmosTableTest.class);
+
+    private static String uri = CosmosTableTestUtils.resolveBaseUri();
+    private static final String key = CosmosTableTestUtils.resolveMasterKey();
+    private static final String database = CosmosTableTestUtils.resolveDatabase();
+
+    @BeforeClass
+    public void init() {
+        log.info("== CosmosDB Collection DELETE tests started ==");
+    }
+
+    @AfterClass
+    public void shutdown() {
+        log.info("== CosmosDB Collection DELETE tests completed ==");
+    }
+
+    @Test
+    public void deleteFromCosmosTableTest1() throws InterruptedException {
+        log.info("deleteFromCosmosTableTest1 - " +
+                "Delete an event of a CosmosDB table successfully");
+
+        String collectionLink = String.format("/dbs/%s/colls/%s", database, "FooTable");
+        CosmosTableTestUtils.dropCollection(uri, key, collectionLink);
+
```

Comment:
@Tikiri16 shall we remove those extra spaces in all the test files?

Reply:
removed.